app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id', dependent: :destroy
+  has_many :comments, foreign_key: 'user_id', dependent: :destroy
+  has_many :likes, foreign_key: 'user_id', dependent: :destroy
+
+  def recent_posts
+    Post.where(author_id: id).order(created_at: :desc).limit(3)
```

Comment:
:orange_circle: [OPTIONAL] Below is a more concise and clear way to achieve the same result as the previous code: 
```
def recent_posts
    posts.order(created_at: :desc).limit(3)
 end
```

 This leverages ActiveRecord associations, using  the  has_many :posts association set up in your this model.